operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -423,57 +426,80 @@ new customized image.
 
 1. (`ncn-mw#`) Update boot parameters for a management node.
 
-    Perform the following procedure **for each xname** identified in the previous step.
-
-    1. Get the existing `metal.server` setting for the component name (xname) of the node of interest.
+    1. When working with a group of `XNAMES` (use the previous step to collect the group of `XNAMES`).
```

Comment:
This shouldn't be a numbered list since you do this option OR the next option, not do them in sequence, if I am understanding the new flow correctly.

```suggestion
    * When working with a group of xnames (use the previous step to collect the xnames in the `$XNAME` variable).
```